pkg/azuredisk/azuredisk_test.go

```diff
@@ -70,102 +69,6 @@ func TestCheckDiskCapacity(t *testing.T) {
 	assert.Equal(t, err, expectedErr)
 }
 
-func TestRun(t *testing.T) {
```

Comment:
test case is removed because I have no clue what scenario is covered here.

Reply:
this is for testing the cloud provider config `azure.json`